### DISCUSSION TITLE
Echanger nos histoires

### DISCUSSION THREAD

**Discussion by gacompere** [marian-m12l/studio#173] [General]
J'ai découvert un super site qui permet d'échanger des histoires créées avec STUdio. Le site permet de poster ses propres histoires et télécharger celles que d'autres studio-nautes ont créées, ou bien les écouter en ligne. J'adore!
C'est par ici:
https://www.mhios.com/fr/

Bonne découverte à tous et au plaisir d'écouter vos histoires!

J'en profite pour dire un grand merci à marian-m12l pour le formidable boulot sur STUdio!!

**Comment by marian-m12l** [comment]
@gacompere Merci pour le retour :-) Si c'est ok pour le créateur du site, j'intégrerai un lien dans le readme (l'outil a été mentionné dans l'issue #129)

**Reply by Djailla** [reply]
Le site partagé n'existe plus :(

**Reply by gacompere** [reply]
@Djailla merci de l'avoir signalé. Je pense qu'il est seulement temporairement offline. Je viens de le signaler à l'administrateur.

**Reply by Djailla** [reply]
Ça fait un petit moment que c'est down. Peut être un problème de droits ?

**Reply by sebbelese** [reply]
Vu la faible fréquentation du site, je n'ai pas renouvelé le nom de domaine. Il est toujours disponible ici:
https://demo-myopenstories.herokuapp.com/

J'avais posté un petit message explicatif sur le forum:
https://demo-myopenstories.herokuapp.com/fr/forum/forum/mhios-9/topic/avenir-du-site-45/

Je pourrais reprendre le nom de domaine, mais je n'avais pas l'impression que ça en valait la peine. Ça demande un peu plus de travail de maintenance, car le domaine en .com envoyait beaucoup de trafic de bots vers le site, ce qui remplissait rapidement la base de données.

**Comment by jersou** [comment]
pour info, il y a un Discord sur ce sujet ici : "Lunii Creations" https://discord.com/channels/911349645752541244/911369446881390592

**Comment by jeromegroucho** [comment]
Bonjour, ni le site internet ni le discord ne semblent fonctionner. Bien a vous, JG

**Comment by jersou** [comment]
Le lien vers le discord : https://discord.com/invite/jg9MjHBWQC il faut lire et accepter le règlement pour accéder au reste.
sinon, pour info, il y avait aussi ce forum avant que le discord ne soit créé : https://www.dealabs.com/discussions/discussion-utilisation-de-studio-pour-lunii-1876791

**Reply by jeromegroucho** [reply]
Merci beaucoup.

**Comment by sebbelese** [comment]
Vu le peu de fréquentation, j'ai arrêté de maintenir le site. Les sources sont dispo si quelqu'un est intéressé:
https://github.com/sebbelese/mhios

**Reply by LeroyErwan** [reply]
Salut, 
Y-a-t-il un moyen de récupérer les histoires qui y étaient hébergées ?

Cdt,

**Reply by sebbelese** [reply]
Oui, les histoires sont accessibles via ce lien:
https://www.dropbox.com/sh/r20dqpz7gru31jt/AAAgoKZCEfEmYr9dVm0tbgGta?dl=0

Il fait zipper le contenu des dossiers "sXX" pour avoir une histoire STUDIO.

Certains dossiers résultent d'uploads ratés et pourraient être incomplets, je n'ai plus la base de données associée.

**Reply by mehdichaouch** [reply]
Hello @sebbelese,

Belle initiative :clap: et beau travail :+1:  que de se mettre au Python (Django) et d'avoir fait [mhios - Mes Histoires Interactive(s) Open Stories](https://github.com/sebbelese/mhios) ([_archive_](https://web.archive.org/web/20210413191812/https://www.mhios.com/fr/)).

J'ai une solution (complètement gratuite) pour relancer le site si vous le souhaitez et avez le temps. Faite le moi savoir en commentaire :wink:

**Reply by sebbelese** [reply]
Bonjour,

Que voulez-vous dire? 

En fait le problème n'était pas vraiment le coût (l'hébergement était gratuit, je ne payais que le nom de domaine ce qui n'était pas énorme), mais plutôt le fait de le maintenir alors qu'il y avait peu d'activité.

**Reply by mehdichaouch** [reply]
@sebbelese si c'est le manque de trafic qui a fait fermer le site, le réouverir ne changera rien, donc laissez tomber ma proposition.
Merci

**Comment by benjapan** [comment]
Un grand merci !

Le dim. 7 août 2022 à 08:57, Sébastien Blaise ***@***.***> a
écrit :
